### PR TITLE
add install instructions

### PR DESCRIPTION
**Pull request by SimeonEhrig**
- We use a local registry. Therefore some extra steps are required beside the default installation commands.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,36 @@
...
+## Installation
+
+To install the current stable version of `QEDevents.jl` you may use the standard julia package manager within the julia REPL
+
+```julia
+julia> using Pkg
+
+# add local registry, where QEDevents is registered
+julia> Pkg.Registry.add(Pkg.RegistrySpec(url="https://github.com/QEDjl-project/registry"))
+# add general registration again, that it is still available beside the local QED registry
```
Comment (szabo137) [inline README.md]: I suggest rephrasing: 

 add general registry again to have it join the local registry

File: README.md [hunk 2]
```diff
@@ -1,3 +1,36 @@
...
+
+julia> Pkg.add("QEDevents")
+```
+
+or you use the Pkg prompt by hitting `]` within the Julia REPL and then type
+
+```julia
+# add local registry, where QEDevents is registered
+(@v1.9) pkg> registry add https://github.com/QEDjl-project/registry
+# add general registration again, that it is still available beside the local QED registry
```
Comment (szabo137) [inline README.md]: I suggest rephrasing: 

add general registry again to have it join the local registry

**Review by szabo137 — CHANGES_REQUESTED** [review]
@SimeonEhrig

File: README.md (hunk 1 above)
Comment (szabo137) [reply README.md]: @SimeonEhrig

File: README.md (hunk 2 above)
Comment (szabo137) [reply README.md]: @SimeonEhrig

**Comment by SimeonEhrig** [comment]
@szabo137 ready to merge

**Review by szabo137 — APPROVED** [review]
Everything is ready to merge!

**Pull request merged**